x-pack/plugin/sql/qa/server/src/main/java/org/elasticsearch/xpack/sql/qa/jdbc/CsvTestUtils.java

```diff
@@ -162,6 +162,8 @@ private static String resolveColumnType(String type) {
                 return "byte";
             case "sh":
                 return "short";
+            case "ul":
+                return "bigdecimal"; // CSV
```

Comment:
The CSV JDBC driver [doesn't support BigInteger](https://github.com/simoc/csvjdbc/blob/2cafaea5d3a5d04ba5df47f11cc9eb4a6e96a080/src/main/java/org/relique/jdbc/csv/StringConverter.java#L646). I've extended the incomplete comment.

Reply:
Shouldn't it be "biginteger" instead of "bigdecimal" though?
As a separate issue, it's considering whether the latest versions of JDBC CVS have added support for this and if not contributing this feature upstream.
It's a good opportunity to check whether an upgrade is warranted, I recall we looked at it in the past but can't remember the details.